docs/Consensus-Protocols/Comparing-PoA.md

```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
 Properties to consider when comparing Clique and IBFT 2.0 are: 
 
 * Immediate finality 
+* Minimum number of validators 
 * Liveness 
 * Speed 
-* Minimum number of validators 
 
 ### Immediate Finality 
 
 IBFT 2.0 has immediate finality. When using IBFT 2.0 there are no forks and all valid blocks are included in the main chain.
 
 Clique does not have immediate finality. Implementations using Clique must be aware of forks and chain reorganizations occurring. 
 
+### Minimum Number of Validators 
+
+IBFT 2.0 requires 4 validators to be Byzantine fault tolerant. 
+
 ### Liveness 
 
 Clique is more fault tolerant than IBFT 2.0. Clique tolerates up to half to the validators failing. IBFT 2.0 networks 
-tolerate up to (n-1)/3 faulty nodes. For example, in an IBFT 2.0 network of:
+require greater than or equal to 2/3 of validators to be operating to create blocks. For example, in an IBFT 2.0 network of:
+
+* 4-5, 1 bad node is tolerated 
+* 6-9, 2 bad nodes are tolerated 
 
-* 3, no bad node are tolerated
-* 4-6, 1 bad node is tolerated 
-* 7-9, 2 bad nodes are tolerated 
+!!! important 
+    An IBFT 2.0 network with 3 validators can tolerate one bad node (that is, continue to produce blocks)
```

Comment:
I'd rephrase this to "can tolerate one unresponsive node (...) but cannot tolerate any malicious nodes". Or Maybe "byzantine" rather than "malicious"

Reply:
Updated